docs/js-api.md

```diff
@@ -70,8 +137,60 @@ Returns a canvas that renders Gosling.js visualization:
 };
 ```
 
-#### options.resolution?: number
-Relative resolution of Gosling.js visualization (Default: `4`).
 
-#### options.transparentBackground?: boolean
-Determine if the background should be transparent or not (Default: `false`).
+
+## subscribe 
+```javascript
+api.subscribe(eventName:string, callback: (msg:string, eventData)=>void)
+```
+Subscribe the callback function to the specified event.
+
+**Parameters**
+
+- **eventName**: string
+Specify the event using its name.
+One of `"mouseOver"`, `"rangeSelect"`, `"click"`, `"rawData"`.
```

Comment:
I just realized that these seem to be based on the last version instead of `0.9.18` (e.g., rangeSelect does not exist in 0.9.18).